ui/app/adapters/kmip/config.js

```diff
@@ -16,9 +16,19 @@ export default BaseAdapter.extend({
     return this._url(...arguments);
   },
   urlForCreateRecord(modelName, snapshot) {
-    return this._url(snapshot.id, modelName, snapshot);
+    const id = snapshot.id || snapshot.record.mutableId;
```

Comment:
I'm wondering if it would be easier to always rely on `mutableId` to be set rather than trying to pull the id from more than one property?

Reply:
Yeah, I was nervous to change too much but I think that works here. It gets more chaotic in the auth methods section